### DISCUSSION TITLE
Add support for `.bun-version`

### DISCUSSION THREAD

**Discussion by sebdanielsson** [renovatebot/renovate#29939] [Suggest an Idea]
### Tell us more.

Similarly to specifying the node version to use the `.node-version` file, a `.bun-version` file can be created to specify the bun version to use. For example, for the official setup-bun [GitHub Action](https://github.com/oven-sh/setup-bun).

Currently, Renovate doesn't support updating this file. Full support for updating my bun environment would be greatly appreciated!

Bun Action: https://github.com/oven-sh/setup-bun
Bun releases: https://github.com/oven-sh/bun/releases

**Comment by rarkins** [comment]
Should it be based off bun tags, or bun releases? I think it should be releases, because those have the binaries attached.

**Reply by sebdanielsson** [reply]
Yeah, using releases is better. I just updated my post to change the link to the releases page.👍

**Comment by rarkins** [comment]
Converted to issue:

- https://github.com/renovatebot/renovate/issues/29941
